src/apps/answers/domain/answers.py

```diff
@@ -639,3 +639,8 @@ class IdentifiersQueryParams(InternalModel):
     respondent_id: uuid.UUID | None = None
     target_subject_id: uuid.UUID | None = None
     answer_id: uuid.UUID | None = None
+
+
+class TakeNowValidationResponse(PublicModel):
+    valid: bool
+    message: str | None = None
```

Comment:
It would be good if validation response contained the details of the `sourceSubject` and `targetSubject` using the `SubjectReadResponse` schema

https://github.com/ChildMindInstitute/mindlogger-backend-refactor/blob/cb980381db9b17a4d4c36fb5407a24614539aa27/src/apps/subjects/domain.py#L82-L86

That way the FE wouldn't need to fetch these after performing this validation (which would eliminate one of the benefits of having a consolidated validation endpoint)

Reply:
This is not an endpoint to get subject detail, its only for validation of take now flow

Reply:
I understand, but part of the justification for creating this endpoint is that the FE would need to make less API calls to get data. If this endpoint doesn't return the data, the FE will need to make all those API calls anyway.

In addition to the ones I requested above, there's also the need for a `currentUserSubject` (based on the designs in [M2-6505](https://mindlogger.atlassian.net/browse/M2-6505)), which is the subject of the authenticated user (using the same `SubjectReadResponse` schema)

Reply:
we should have a call about it @mbanting @sultanofcardio

Reply:
Thanks for the discussion @rcmerlo, @sultanofcardio . As we agreed, we'll move forward as is, given that it
- addresses the main purpose of this ticket and endpoint
- avoids the need to move to a different resource endpoint that would be more appropriate for the payload suggested

However, the suggestion by @sultanofcardio is a valid one and we'll consider a future endpoint to GET a payload needed by multi-informant assessments but also validates the parameters provided. This is inline with other GET calls that perform some validation as part of the service.